src/repeater/repeater.js

```diff
@@ -219,8 +263,12 @@ function createGeneratorRepeater(generator) {
 module.exports = {
   create,
   createGeneratorRepeater,
+  pause,
```

Comment:
are these exports just for testing or are the functions called elsewhere? if just for testing, please add comment as needed

Reply:
They will be called by the pause and resume commands.